src/Akka.Persistence.MongoDb.Hosting.Tests/Akka.Persistence.MongoDb.Hosting.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkTestVersion);$(NetTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
```

Comment:
`$(NetFrameworkTestVersion)` - I did not have the Netframework developer kit installed on my machine so the tests have not been run against framework. Given the changes I don't believe it will be an issue but if required I will get them installed. They were all run against `NetTestVersion` and `NetCoreTestVersion`

Reply:
No worries - looks like things are passing on CI/CD so far. We can probably drop .NET Framework for testing MongoDb in the near future anyway.